net/interfaces/defaultroute_ios.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build ios
+
+package interfaces
+
+import (
+	"log"
+	"sync/atomic"
+)
+
+var (
+	lastKnownDefaultRouteIfName atomic.Value // of string
```

Comment:
FYI that we have [`syncs.AtomicValue`](https://pkg.go.dev/tailscale.com/syncs#AtomicValue) which is a generic version of this

Reply:
Switched! Thanks